### DISCUSSION TITLE
I should be back soon

### DISCUSSION THREAD

**Discussion by zHaytam** [Blazor-Diagrams/Blazor.Diagrams#452] [Announcements]
Hello,

It's been a while since anything happened in this repository, and I know people are starting to wonder if I've abandoned it.

These last weeks/months have been mentally draining, to be honest. From one of my close relatives dying, my dad going to the hospital, and me dealing with my own "annual" health issues.

Thankfully, everything is getting better now, and I'm just settling and dealing with some work stuff. I'll be back to taking care of everything in this repository very soon!

**Comment by Discrete-Functions** [comment]
I am sorry to hear that you have had so many personal challenges. Hopefully things will soon normalize for you.

**Comment by qallpat** [comment]
Sorry to hear about that. 

It is a very good product you've created here.
